package-lock.json

```diff
@@ -2128,7 +2128,7 @@
         },
         "strip-ansi": {
           "version": "3.0.1",
-          "resolved": "https://registry.npmjs.org/strip-ansi/-/strip-ansi-3.0.1.tgz",
+          "resolved": "http://registry.npmjs.org/strip-ansi/-/strip-ansi-3.0.1.tgz",
```

Comment:
One thing, could you — this one time — run:

```
npm cache clear --force
```

and then run `npm install` again and see if it changes these back to `https`?